### PR TITLE
feat(GDB-11205): Refactor and Update Configuration and Scripts

### PR DESCRIPTION
**Pull request by teodossidossev**
## WHAT
Refactored the ontotext-graphql-playground project by introducing new scripts, updating configurations, and enhancing code quality rules. Removed obsolete files and adjusted project dependencies.

## WHY
- To update cleaning, building, and linting processes.
- Align project configurations with organizations standards and remove unnecessary settings and dependencies.
- Enforce stricter linting rules and improve error handling in scripts
- Simplify asset management and ensure consistency in build processes
- Introduced copy-files.sh, a script to streamline asset management during the build process

## HOW
Scripts:
- Added scripts/clean.sh and scripts/error-handling.sh for cleanup and error handling.
- Introduced copy-files.sh for managing assets during the build process
- Deleted .prettierrc.json as it is no longer needed. Linting Rules:
- Enhanced .eslintrc.json with stricter TypeScript and React rules Dependencies:
- Updated package.json files to include @typescript-eslint and eslint-plugin-react Code Changes:
- Refined methods in graphql-playground-component.tsx to include proper TypeScript return types
- Removed redundant logs (console.log) Build Process:
- Updated package.json scripts to manage dependencies, build tasks, and linting
- Updated DEVELOPMENT.md

### CODE REVIEW

**Comment by teodossidossev** [comment]
Jenkins retry build

**Comment by teodossidossev** [comment]
Jenkins retry build

**Comment by teodossidossev** [comment]
Jenkins retry build

**Comment by hudson-onto** [comment]
Build failed!

**Comment by teodossidossev** [comment]
Jenkins retry build

**Comment by hudson-onto** [comment]
Build failed!

**Comment by teodossidossev** [comment]
Jenkins retry build

File: .idea/misc.xml
Comment (svilenvelikov) [inline .idea/misc.xml:1]: .idea files must be ignored

File: ontotext-graphql-playground-component/.eslintrc.json
Comment (svilenvelikov) [inline ontotext-graphql-playground-component/.eslintrc.json:1]: This is the old eslint format. Why don't you update to the latest version

**Comment by svilenvelikov** [comment]
I'd preffer the jenkins job to be in the graphdb tab.

File: .idea/misc.xml
Comment (teodossidossev) [reply .idea/misc.xml:1]: yes, I'm deleting the files; added folder to gitignore

File: ontotext-graphql-playground-component/.eslintrc.json
Comment (teodossidossev) [reply ontotext-graphql-playground-component/.eslintrc.json:1]: updated

**Comment by teodossidossev** [comment]
> I'd preffer the jenkins job to be in the graphdb tab.

it is already there

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**